tests/foreman/api/test_contentmanagement.py

```diff
@@ -562,68 +562,68 @@ def test_positive_capsule_sync(self):
-        """Ensure RH repo with ISOs after publishing to Library is synchronized
-        to capsule automatically
-
-        :id: 221a2d41-0fef-46dd-a804-fdedd7187163
-
-        :customerscenario: true
-
-        :BZ: 1303102, 1480358, 1303103, 1734312
-
-        :expectedresults: ISOs are present on external capsule
-
-        :CaseLevel: System
-        """
-        # Create organization, product, enable & sync RH repository with ISOs
-        org = entities.Organization(smart_proxy=[self.capsule_id]).create()
-        with manifests.clone() as manifest:
-            upload_manifest(org.id, manifest.content)
-        rh_repo_id = enable_rhrepo_and_fetchid(
-            basearch='x86_64',
-            org_id=org.id,
-            product=PRDS['rhsc'],
-            repo=REPOS['rhsc7_iso']['name'],
-            reposet=REPOSET['rhsc7_iso'],
-            releasever=None,
-        )
-        rh_repo = entities.Repository(id=rh_repo_id).read()
-        call_entity_method_with_timeout(rh_repo.sync, timeout=2500)
-        capsule = entities.Capsule(id=self.capsule_id).read()
-        # Find "Library" lifecycle env for specific organization
-        lce = entities.LifecycleEnvironment(organization=org).search(
-            query={'search': 'name={}'.format(ENVIRONMENT)}
-        )[0]
-        # Associate the lifecycle environment with the capsule
-        capsule.content_add_lifecycle_environment(data={'environment_id': lce.id})
-        result = capsule.content_lifecycle_environments()
-        self.assertGreaterEqual(len(result['results']), 1)
-        self.assertIn(lce.id, [capsule_lce['id'] for capsule_lce in result['results']])
-        # Create a content view with the repository
-        cv = entities.ContentView(organization=org, repository=[rh_repo]).create()
-        # Publish new version of the content view
-        cv.publish()
-        cv = cv.read()
-        self.assertEqual(len(cv.version), 1)
-        # Verify ISOs are present on satellite
-        repo_path = os.path.join(PULP_PUBLISHED_ISO_REPOS_PATH, rh_repo.backend_identifier)
-        sat_isos = get_repo_files(repo_path, extension='iso')
-        self.assertGreater(len(result), 0)
-        # Assert that a task to sync lifecycle environment to the capsule
-        # is started (or finished already)
-        sync_status = capsule.content_get_sync()
-        self.assertTrue(
-            len(sync_status['active_sync_tasks']) >= 1 or sync_status['last_sync_time']
-        )
-        # Wait till capsule sync finishes
-        for task in sync_status['active_sync_tasks']:
-            entities.ForemanTask(id=task['id']).poll(timeout=600)
-        # Verify all the ISOs are present on capsule
-        capsule_isos = get_repo_files(repo_path, extension='iso', hostname=self.capsule_ip)
-        self.assertGreater(len(result), 0)
-        self.assertEqual(set(sat_isos), set(capsule_isos))
+    # @tier4
+    # def test_positive_iso_library_sync(self):
```

Comment:
Wouldn't it be better to just add `@stubbed` decorator?

As far as I know, `@stubbed` still end up in Polarion. I'm not versed enough in DDR/PQI to tell what exact impact that has, but I heard we generally don't want things removed from Polarion once they appear there.

Reply:
using @stubbed as @JacobCallahan suggested.